homeassistant/components/rejseplanen/sensor.py

```diff
@@ -109,14 +109,24 @@ def state(self):
     def device_state_attributes(self):
         """Return the state attributes."""
         if not self._times:
-            return None
-
-        next_up = []
+            return {
+                ATTR_DUE_IN: None,
```

Comment:
Our standard is to not include the item at all if the value is None.

Reply:
Even if the item isn't always None? Wouldn't that cause any templates using those attributes to throw errors?

Reply:
The templates should check for that and handle it.